dictionary/README.md

```diff
@@ -1,5 +1,5 @@
+I originally picked the permanent restart strategy for my dictionary as I thought we should always have a dictionary process running when we plan on accessing the random_word calls.  This way we can ensure that we always have access to this call.  Also, this process seems very unlikely to crash so in the case that it does for some reason (likely some sort of edge case use) we should simply reboot it - it seems unlikely that some sort of malicious user will be able to frequently crash this process so I see no reason to not reboot it when it crashes.
```

Comment:
Given that you argue both sides, I'm happy to go with it, because it shows you thought it through.